up-streamer/src/ustreamer.rs

```diff
@@ -410,6 +452,10 @@ pub struct UStreamer {
     registered_forwarding_rules: ForwardingRules,
     transport_forwarders: TransportForwarders,
     forwarding_listeners: ForwardingListeners,
+    subscription_cache: Arc<Mutex<SubscriptionCache>>,
+    // TODO: Use this when USubsription is implemented
```

Comment:
I think usubscription here is not needed, now, nor in the future, since we'd be making Request UMessages to send out over the host transport, not using a USubscription trait object directly.

Reply:
removed it